src/stories/containers/finances-overview/Components/ExpensesChart/ExpensesChart.stories.tsx

```diff
@@ -0,0 +1,291 @@
+        },
+      },
+      {
+        value: new TotalExpenseReportsBuilder().withPrediction(0).build().prediction,
+        itemStyle: {
+          borderRadius: [0, 0, 6, 6],
+        },
+      },
+      {
+        value: new TotalExpenseReportsBuilder().withPrediction(0).build().prediction,
+        itemStyle: {
+          borderRadius: [0, 0, 6, 6],
+        },
+      },
+      {
+        value: new TotalExpenseReportsBuilder().withPrediction(0).build().prediction,
+        itemStyle: {
+          borderRadius: [0, 0, 6, 6],
+        },
+      },
+
+      {
+        value: new TotalExpenseReportsBuilder().withPrediction(0).build().prediction,
+        itemStyle: {
+          borderRadius: [0, 0, 6, 6],
+        },
+      },
+      {
+        value: new TotalExpenseReportsBuilder().withPrediction(0).build().prediction,
+        itemStyle: {
+          borderRadius: [0, 0, 6, 6],
+        },
+      },
+      {
+        value: new TotalExpenseReportsBuilder().withPrediction(0).build().prediction,
+        itemStyle: {
+          borderRadius: [0, 0, 6, 6],
+        },
+      },
+      {
+        value: new TotalExpenseReportsBuilder().withPrediction(0).build().prediction,
+        itemStyle: {
+          borderRadius: [0, 0, 6, 6],
+        },
+      },
+      {
+        value: new TotalExpenseReportsBuilder().withPrediction(1000000).build().prediction,
+        itemStyle: {
+          borderRadius: [6, 6, 0, 0],
+        },
+      },
+      {
+        value: new TotalExpenseReportsBuilder().withPrediction(1600000).build().prediction,
+        itemStyle: {
+          borderRadius: [6, 6, 0, 0],
+        },
+      },
+      {
+        value: new TotalExpenseReportsBuilder().withPrediction(1370000).build().prediction,
+        itemStyle: {
+          borderRadius: [6, 6, 0, 0],
+        },
+      },
+      {
+        value: new TotalExpenseReportsBuilder().withPrediction(900000).build().prediction,
+        itemStyle: {
+          borderRadius: [6, 6, 0, 0],
+        },
+      },
+    ],
+  },
+];
+
+export const [[LightMode, DarkMode]] = createThemeModeVariants(ExpensesChart, args);
+LightMode.parameters = {
+  figma: {
+    component: {
+      375: {
+        component:
+          'https://www.figma.com/file/pyaYEjcwF2b5uf9y0vIfIy/SES-Dashboard?node-id=13056%3A123466&t=CFEpU7Gadr2fxQ94-4',
+        options: {
+          componentStyle: {
+            width: 343,
+            height: 265,
+          },
+          style: {
+            top: 0,
+            left: -1,
+            width: 342,
+            height: 260,
+          },
+        },
+      },
+      834: {
+        component:
+          'https://www.figma.com/file/pyaYEjcwF2b5uf9y0vIfIy/SES-Dashboard?node-id=13562%3A143904&t=3sj8JoMArfel9d4j-4',
+        options: {
+          componentStyle: {
+            width: 607,
+            height: 387,
+          },
+          style: {
+            top: -2,
+            left: 3,
+            width: 607,
+            height: 382,
```

Comment:
The `options > style` styles are for the Figma image container, it is now a good idea to set `width` and `height` here as the image should have the intrinsic size